packages/angular/src/migrations/update-13-3-0/update-mfe-webpack-config.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ProjectConfiguration,
+  Tree,
+  updateProjectConfiguration,
+} from '@nrwl/devkit';
+import { logger, formatFiles, getProjects } from '@nrwl/devkit';
+import { tsquery } from '@phenomnomnominal/tsquery';
+import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { Node } from 'typescript';
+
+export default async function (tree: Tree) {
+  const NRWL_WEBPACK_BROWSER_BUILDER = '@nrwl/angular:webpack-browser';
+  const CUSTOM_WEBPACK_OPTION = 'customWebpackConfig';
+
+  const projects = getProjects(tree);
+
+  const projectsToUpdateServeTarget = new Map<string, ProjectConfiguration>();
+  forEachExecutorOptions(
+    tree,
+    NRWL_WEBPACK_BROWSER_BUILDER,
+    (opts, projectName) => {
+      // Update the webpack config
+      const webpackPath = opts[CUSTOM_WEBPACK_OPTION].path;
+      if (!tree.exists(webpackPath)) {
+        logger.warn(
+          `Webpack config file for project: ${projectName} does not exist. Skipping project.`
+        );
+        return;
+      }
+      const webpackConfig = tree.read(webpackPath, 'utf-8');
+      if (!webpackConfig.includes('ModuleFederationPlugin')) {
+        logger.warn(
+          `Webpack config file for project: ${projectName} is not using Module Federation. Skipping project.`
+        );
+        return;
+      }
+
+      const updatedWebpackFile = updateScriptType(webpackConfig);
+      tree.write(webpackPath, updatedWebpackFile);
+
+      // migrate the serve-mfe target to use liveReload=false
+      const project = projects.get(projectName);
+      if ('serve-mfe' in project.targets) {
+        projectsToUpdateServeTarget.set(projectName, project);
+      }
+    }
+  );
+
+  for (const [projectName, project] of projectsToUpdateServeTarget) {
+    const updatedCommands = project.targets['serve-mfe'].options.commands.map(
+      (command: string) => `${command} --liveReload=false`
```

Comment:
I think we should handle the scenario where the devs have already applied the workaround. We have already communicated what the workaround is, by the time this gets released some workspaces would have it and in that case, this would cause the `--liveReload=false` to appear twice.

Reply:
True, good catch